.eslintrc.js

```diff
@@ -255,7 +259,7 @@ module.exports = {
         "operator-linebreak": "error",
         "padded-blocks": "off",
         "padding-line-between-statements": "error",
-        "prefer-arrow-callback": "error",
+        "prefer-arrow-callback": "off",
```

Comment:
Why are we modifying those rules? I think we use the same setup in multiple repos, it'd be good to keep the same standard.

Reply:
It was mainly to fix issues in the mocha test file. I reverted these changes and made exceptions inside the file itself.